src/spikeinterface/preprocessing/preprocessinglist.py

```diff
@@ -54,6 +56,7 @@
     ScaleRecording,
     CenterRecording,
     ZScoreRecording,
+    ScaleTouV,
```

Comment:
Everything else has Recording in the name. Are you just against that?

Reply:
Not really but I feel the patttern does not fit. 
ScaleRecoding : ScaleTheRecording 
ZScoreRecording: ZScoreTheRecording
CenterRecording: CenterTheRecording

Pattern: Verb(To)Recording

I think the pattern should be (Adective)Recording  like ScaledRecording, CenteredRecording but I guess that shiped has sailed. 

Would you like me to add Recording? I have the vague feeling that is marginally better to be consistent.

Reply:
It is just hard for me to visually parse `ScaleTouV` because micro is the first word so it should be `ScaleToUV`, but then you have the double capital which is also tricky. I'm just hoping that `ScaleTouVRecording` feels better, but it is still a little tricky for me to view. I don't really have a solution, but consistency like you said at least gives us a marginal benefit.

Reply:
Done.